### PR TITLE
fix: fix overflow bug

### PR DESCRIPTION
**Pull request by rgomeztao**
coming from [AUT-3027](https://oat-sa.atlassian.net/browse/AUT-3027)

changelog:
* fixes overflow on associate interaction
* if choice contains mathjax, it will stretch
* images will now be limited to 180px max-width by css

notes:
* confirmed change  doesnt affect the visualizartion in next gen previewer
image

[AUT-3027]: https://oat-sa.atlassian.net/browse/AUT-3027?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jsconan — CHANGES_REQUESTED** [review]
The issue described in the ticket is fixed.
However, it also introduces another issue as a choice with content that fits the container cannot be dragged and dropped in any box.

image

<video src="https://user-images.githubusercontent.com/1500098/229710666-5a7227dd-544d-454b-ac9a-efdc561af63c.mp4"></video>

Regarding the PR itself, I also see 2 things that are not correct:
- the base branch should be `develop` not `master`, unless it is a release branch.
- the branch name does not reflect the intent: I don't think it is a feature as it is rather a fix

File: views/scss/qti/_associate.scss [hunk 1]
```diff
@@ -18,7 +18,6 @@
         & > .qti-choice {
             min-width: 100px;
             min-height: 30px;
-            max-width: 180px;
```
Comment (jsconan) [inline views/scss/qti/_associate.scss:21]: Quick fix :)
But I wonder if it is not too aggressive, see my previous comment.

BTW, this line has not changed for 7y, the last change is from https://github.com/oat-sa/extension-tao-itemqti/commit/978c2f165e83e8f2e285df879a1c453402719234#diff-f2f402bcd0bcb674fae83ec4cd965a7c21527cb011203c050271d161c2a42ab6L21

**Review by jsconan — COMMENTED** [review]
It works, but it looks a bit odd. Maybe should we add a word-wrap?
image

BTW, I don't see the rule for the choices having a Maths expression

**Comment by rgomeztao** [comment]
added word wrap for long text strings :)

you can find the selector for the math expressions here
image

thanks! :)

**Review by jsconan — APPROVED** [review]
The issue is fixed. Nice job!

image

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code respects code style rules
- [x] New code respects best practices
- [ ] ~~New code is not subject to concurrency issues (if applicable)~~
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic

**Review by jsconan — APPROVED** [review]

**Pull request merged**